kurtosis-devnet/cmd/main.go

```diff
@@ -271,20 +297,66 @@ func (m *Main) deploy(ctx context.Context, r io.Reader) error {
 		return fmt.Errorf("error creating kurtosis deployer: %w", err)
 	}
 
-	env, err := d.Deploy(ctx, buf)
+	spec, err := d.Deploy(ctx, buf)
 	if err != nil {
 		return fmt.Errorf("error deploying kurtosis package: %w", err)
 	}
 
+	env, err := d.GetEnvironmentInfo(ctx, spec)
+	if err != nil {
+		return fmt.Errorf("error getting environment: %w", err)
+	}
+
 	if err := writeEnvironment(m.cfg.environment, env); err != nil {
 		return fmt.Errorf("error writing environment: %w", err)
 	}
 
 	return nil
 }
 
+func (m *Main) deployFileserver(ctx context.Context, sourceDir string) error {
+	// Create a temp dir in the fileserver package
+	baseDir := filepath.Join(m.cfg.baseDir, "fileserver")
+	if err := os.MkdirAll(baseDir, 0755); err != nil {
+		return fmt.Errorf("error creating base directory: %w", err)
+	}
+	tempDir, err := os.MkdirTemp(baseDir, "fileserver-content")
+	if err != nil {
+		return fmt.Errorf("error creating temporary directory: %w", err)
+	}
+	defer os.RemoveAll(tempDir)
+
+	// Copy build dir contents to tempDir
+	if err := util.CopyDir(sourceDir, tempDir); err != nil {
+		return fmt.Errorf("error copying directory: %w", err)
+	}
+
+	buf := bytes.NewBuffer(nil)
+	buf.WriteString(fmt.Sprintf("source_path: %s\n", filepath.Base(tempDir)))
+
+	opts := []kurtosis.KurtosisDeployerOptions{
+		kurtosis.WithKurtosisBaseDir(m.cfg.baseDir),
+		kurtosis.WithKurtosisDryRun(m.cfg.dryRun),
+		kurtosis.WithKurtosisPackageName("fileserver/"),
```

Comment:
nit: maybe extract that constant near top of file, to make it easier to find if/when we refator ?